src/style.css

```diff
@@ -1 +1,57 @@
 /* add styles here */
+
+.body {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.all__stars {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  background-image: url('/src/images/star.svg');
```

Comment:
Remember to use quotes for the URL in the `background-image` property for consistency and to avoid potential issues with certain characters or spaces in the path.